WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -351,6 +351,34 @@ final class OrderListViewModel {
     }
 }
 
+// MARK: - In-Person Payments Feedback Banner
+
+extension OrderListViewModel {
+    func dismissIPPFeedbackBanner(remindAfterDays: Int?) {
```

Comment:
Nice change moving the logic to the view model! However, the `remindAfterDays` logic can be contained in the view model; the view doesn't need to know about when are we going to show it again. With that, we can remove this parameter `remindAfterDays` from this function.

Reply:
Changed here the remaining use of `remindAfterDays`: https://github.com/woocommerce/woocommerce-ios/pull/8657/commits/d5e02a27e6a954cb33fb0ed65709996599de07b5 to a new method, so this is entirely contained in the view model now 👍